### PR TITLE
[enhancement] Added Sailor trainer class

### PR DESCRIPTION
**Pull request by Alekemon**
3rd out of the 4 trainer class PRs, this trainer uses Water and Fighting type Pokemon

### CODE REVIEW

**Comment by CodeTappert** [comment]
German: "Matrose"

**Comment by bennybroseph** [comment]
Is dialogue being added for this as well?

**Comment by Alekemon** [comment]
> Is dialogue being added for this as well?

I added some lines from the sailors in Kanto games

**Comment by Alekemon** [comment]
Captura de pantalla 2024-06-06 141530

Screenshot

**Comment by Alekemon** [comment]
Captura de pantalla 2024-06-06 173055

Properly centered screenshot

File: public/images/sailor.png
Comment (bennybroseph) [inline public/images/sailor.png:1]: why is there two

**Review by bennybroseph — CHANGES_REQUESTED** [review]

File: public/images/sailor.png
Comment (Alekemon) [reply public/images/sailor.png:1]: First screenshot showed an earlier build of the PR which was readjusted in the second screenshot, I deleted the first screenshot and left the final one

Comment (Alekemon) [reply public/images/sailor.png:1]: Oh you didn't mean the screenshot, let me take a look

File: src/locales/de/dialogue.ts [hunk 1]
```diff
@@ -379,6 +379,18 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+  "sailor": {
+    "encounter": {
+      1: "Matey, you're walking the plank if you lose!",
+      2: "Come on, then! My sailor's pride is at stake!",
+      3: "Ahoy, there! Are you seasick?"
+    },
+    "victory": {
+      1: "Argh! Beaten by a kid!",
+      2: "Your spirit sank me!",
+      3: "I was just careless!"
```
Comment (bennybroseph) [inline src/locales/de/dialogue.ts:391]: I think it's supposed to be "Come on then!" with no comma. Same with "Ahoy there!"

Would also like:
- "I was just careless!" changed to "I think it's me that's seasick..."

**Review by bennybroseph — CHANGES_REQUESTED** [review]

File: src/locales/de/dialogue.ts (hunk 1 above)
Comment (Alekemon) [reply src/locales/de/dialogue.ts]: Ok, I applied these changes, thanks for clarifying!

**Comment by bennybroseph** [comment]
what are you using to version these? This seems crazy 
image

**Comment by Alekemon** [comment]
> what are you using to version these? This seems crazy image

I just copy and paste the string over and over for each language available

**Comment by bennybroseph** [comment]
you don't have to make a commit for each file brother

**Comment by bennybroseph** [comment]
Same for these, idk how that happened
image

**Review by bennybroseph — APPROVED** [review]

**Comment by Alekemon** [comment]
> you don't have to make a commit for each file brother

Ok, I'll note that. Should I revert those commits?

**Pull request merged**